input/pagecontent/StructureDefinition-ShareableViewDefinition-notes.md

```diff
@@ -0,0 +1,30 @@
+### Required FHIRPath Expressions/Functions
+
+All View Runners claiming conformance to the Shareable View Definition profile must implement these [FHIRPath](https://hl7.org/fhirpath/)
+capabilities:
+
+-   [Literals](https://hl7.org/fhirpath/#literals) for String, Integer and Decimal
+-   [where](https://hl7.org/fhirpath/#wherecriteria-expression-collection)function
+-   [exists](https://hl7.org/fhirpath/#existscriteria-expression-boolean) function
+-   [empty](https://hl7.org/fhirpath/#empty-boolean) function
+-   [extension](https://hl7.org/fhir/R4/fhirpath.html#functions) function
+-   [ofType](https://hl7.org/fhirpath/#oftypetype-type-specifier-collection)
+    function
+-   [first](https://hl7.org/fhirpath/#first-collection) function
+-   Boolean
+    operators: [and](https://hl7.org/fhirpath/#and), [or](https://hl7.org/fhirpath/#or), [not](https://hl7.org/fhirpath/#not-boolean)
+-   Math
+    operators: [addition (+)](https://hl7.org/fhirpath/#addition), [subtraction (-)](https://hl7.org/fhirpath/#subtraction), [multiplication (\*)](https://hl7.org/fhirpath/#multiplication), [division (/)](https://hl7.org/fhirpath/#division)
+-   Comparison
+    operators: [equals (=)](https://hl7.org/fhirpath/#equals), [not equals (!=)](https://hl7.org/fhirpath/#not-equals), [greater than (>)](https://hl7.org/fhirpath/#greater-than), [less or equal (<=)](https://hl7.org/fhirpath/#less-or-equal)
+-   [Indexer expressions](https://hl7.org/fhirpath/#index-integer-collection)
+
+### Experimental FHIRPath Functions
```

Comment:
I'm not quite sure how to do a "big yellow box", but maybe this is good enough for now.